editor/src/clj/editor/scene_tools.clj

```diff
@@ -460,26 +460,22 @@
       (let [world-transform (:world-transform (peek original-values))
             start-delta (doto (Vector3d.) (.sub start-pos manip-pos))
             delta (doto (Vector3d.) (.sub pos manip-pos))
-            axis-scale (fn [axis]
-                         (let [local (case axis
-                                       :x (Vector3d. 1.0 0.0 0.0)
-                                       :y (Vector3d. 0.0 1.0 0.0)
-                                       :z (Vector3d. 0.0 0.0 1.0))]
-                           (.normalize local)
-                           (.transform ^Matrix4d world-transform local)
-                           (let [start-len (.dot start-delta local)
-                                 curr-len (.dot delta local)]
-                             (if (> (Math/abs start-len) 1e-12)
-                               (/ curr-len start-len)
-                               1.0))))
+            axis-scale (fn [local]
```

Comment:
Doesn't this give reflection errors unless you tag `^Vector3d local`? 🤔

Reply:
I aalways forget it =/